qdata/parser.py

```diff
@@ -302,14 +302,15 @@ def qop(self, *args):
         if len(args) == 1:
             # <uop>
             return args[0]
-
         if args[0] == "reset":
             # reset <argument>;
-            wires = args[1]
+            # `Op` takes a list of wires while `args[1]`` is of type `['q', 0]``
```

Comment:
```suggestion
            # `Op` takes a list of wires while `args[1]`` is of type `['q', 0]`
```

Reply:
Oh, right, thanks for reminding me. :smile: